src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package wolox.training.models;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import wolox.training.constants.ExceptionsConstants;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE)
+	private int id;
+
+	@NotNull(message = "User username cannot be null")
+	@Column(nullable = false)
+	private String username;
+
+	@NotNull(message = "User name cannot be null")
+	@Column(nullable = false)
+	private String name;
+
+	@NotNull(message = "User birthdate cannot be null")
+	@Column(nullable = false)
+	private Date birthdate;
+
+	@ManyToMany
+	@JoinTable(
+			name = "user_book",
+			joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+			inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id")
+	)
```

Comment:
`Join Table` is not necessary, JPA will do this automatically

Reply:
Done!